microsoft/testsuites/core/provisioning.py

```diff
@@ -89,3 +70,73 @@ def smoke_test(self, case_name: str, node: RemoteNode) -> None:
             ):
                 raise LisaException(f"after reboot, {identifier}")
             raise PassedException(identifier)
+
+    @TestCaseMetadata(
+        description="""
+        This case verifies whether a node is operating normally.
+
+        Steps,
+        1. Connect to TCP port 22. If it's not connectable, failed and check whether
+            there is kernel panic.
+        2. Connect to SSH port 22, and reboot the node. If there is an error and kernel
+            panic, fail the case. If it's not connectable, also fail the case.
+        3. If there is another error, but not kernel panic or tcp connection, pass with
+            warning.
+        4. Otherwise, fully passed.
+        """,
+        priority=0,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole, DiskStandardLRS],
+        ),
+    )
+    def smoke_test(self, case_name: str, node: RemoteNode) -> None:
+        self._smoke_test(case_name, node)
+
+    @TestCaseMetadata(
+        description="""
+        This case verifies whether a node is operating normally.
+
+        Steps,
```

Comment:
It can explain that refer to the `smoke_test`, it doesn't need to repeat content here.